### ISSUE TITLE
ssh.process(..., cwd=) is broken

### ISSUE THREAD

**Issue by zachriggle** [Gallopsled/pwntools#1479]
It looks like `cwd` is always set to `'.'` regardless of what the actual working directory is.

This breaks fetching core files with `CorefileFinder`.

```py
s=ssh('level1', 'io.netgarage.org', password='level1')

# set_working_directory creates a temporary working directory for all follow-up processes
s.set_working_directory()
# [*] Working directory: '/tmp/tmp.Nq3ANxhDBR'
io = s.process(['pwd'])
io.recvall()
# b'/tmp/tmp.Nq3ANxhDBR\n'
io.cwd
# '.'

# cwd= specifies a directory to work out of
io = s.process(['pwd'], cwd='/tmp')
io.recvall()
# b'/tmp\n'
io.cwd
# '.'
```

**Comment by zachriggle** [comment]
It looks like we do not pass `cwd=` to the `ssh_process()` invocation.  Separately, the variable is named `wd` for `ssh_channel`.  I'm in favor of renaming the `wd=` to `cwd=`, since it's unlikely anybody ever manually invoked `ssh_channel` or `ssh_process`.

**Comment by Arusekk** [comment]
I think this is kind of fixed on current dev, please check

**Comment by peace-maker** [comment]
Fixed in #2241 and #2251
